gradle.properties

```diff
@@ -1,6 +1,6 @@
 kotlin.code.style=official
 kotlin_version=1.4.10
-release_version=0.0.14
+release_version=0.0.15
```

Comment:
Mayber better to up minor version

Reply:
We haven't added new features or extended existing ones